src/lib/builder/mint-template-builder.ts

```diff
@@ -201,3 +201,5 @@ export class MintTemplateBuilder {
     return this;
   }
 }
+
+export const DATE_DISTANT_FUTURE = new Date(2524608000);
```

Comment:
This date is in seconds, not MS which resolves to 1/30/1970.

Should be `new Date(2524607000000);` for JS/TS